controllers/topology/util.go

```diff
@@ -114,18 +114,31 @@ func (l *topologyReconcileLogger) Into(ctx context.Context) (context.Context, lo
 	return ctrl.LoggerInto(ctx, l.Logger), l
 }
 
-// KRef return a reference to a Kubernetes object in the same format used by kubectl commands (kind/name).
-type KRef struct {
+// KObj return a reference to a Kubernetes object in the same format used by kubectl commands (kind/name).
+// Note: We're intentionally not using klog.KObj as we want the kind/name format instead of namespace/name.
+type KObj struct {
```

Comment:
Should this be exported?

Reply:
I think we should move it into an internal package. Otherwise it cannot be used by other packages in topology/* (either because it's not exported or because of circular dependencies)

Reply:
I'm +1 to keep this internal at this stage (no preferences if this should be in separated package).

Reply:
Done, moved to internal/log